sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/spark.py

```diff
@@ -186,7 +185,9 @@ def __init__(
 
         is_spark_component = isinstance(component, SparkComponent)
```

Comment:
NIT: only referred once? if so, we may use `if isinstance(component, SparkComponent):` directly

Reply:
code in line 215 also refer